README.md

```diff
@@ -26,6 +26,7 @@ docker run --rm -d \
 docker run -d \
 	--network icinga \
 	--restart always \
+        --name icingadb \
 	-e ICINGADB_REDIS_HOST=redis-icingadb \
```

Comment:
This mixes indentation. The surrounding lines are indented with a tab while the newly added line is indented with spaces (in the other place as well).

Reply:
Good spot! I've updated the PR with a new commit that resolves this.